packages/react-headless/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @magicbell/react-headless
 
+## 4.2.4
+
+### Patch Changes
+
+- [#47](https://github.com/magicbell-io/magicbell-js/pull/47) [`8d30258`](https://github.com/magicbell-io/magicbell-js/commit/8d302586175a1219c743b0135038538a591e0a0c) Thanks [@smeijer](https://github.com/smeijer)! - fix: initialize stores in an effect instead of lazy use-state.
```

Comment:
```suggestion
- [#43](https://github.com/magicbell-io/magicbell-js/pull/43) [`d9d2318`](https://github.com/magicbell-io/magicbell-js/commit/d9d23180be66f0487b12c71440eb1cf1bbcb41c9) Thanks [@rollacaster](https://github.com/rollacaster)! - fix: initialize stores in an effect instead of lazy use-state.
```